articles/storage/blobs/data-lake-storage-acl-powershell.md

```diff
@@ -263,6 +263,7 @@ Update-AzDataLakeGen2AclRecursive -Context $ctx -FileSystem $filesystemName -Pat
 
 > [!NOTE]
 > To a set the ACL of a specific group or user, use their respective object IDs. For example, `group:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx` or `user:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`.
+++To set the ACL of a specific group or user, service principal, managed identity use their respective object IDs. For example, to set the ACL of a **group**, use `group:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`. To set the ACL of a **user**, use `user:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`.
```

Comment:
```suggestion
> To set the ACL of a specific group or user, service principal, or managed identity, use their respective object IDs. For example, to set the ACL of a **group**, use `group:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`. To set the ACL of a **user**, use `user:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`.
```
Minor edits to fix grammar and formatting. But note that this sentence is nearly identical to line 265. Was it intended to replace line 265?

Reply:
Yes please. Wow I didn't notice that the initial PR included an additional paragraph. It should just be a modification to the existing paragraph which adds service principal and managed identity to the comma separated list. Thanks for catching that!